eox_tenant/signals.py

```diff
@@ -217,3 +225,97 @@ def clear_tenant(sender, request, **kwargs):  # pylint: disable=unused-argument
     Signal: django.core.signals.got_request_exception
     """
     pass
+
+
+def before_task_publish(sender, *args, **kwargs):  # pylint: disable=unused-argument
+    """
+    Dispatched before a celery task is published. Note that this is executed in the process sending the task.
```

Comment:
This should explain the reasoning for this function so that a human can understand the purpose of the function. Also, as the previous comment noted, this is the receiver of the signal.